Gemfile

```diff
@@ -25,7 +25,7 @@ gem 'nokogiri'
 
 gem 'responders'
 
-gem 'rack-timeout'
+gem 'rack-timeout', require: "rack/timeout/base"
```

Comment:
In order to configure the service_timeout, this change is required for manual middleware initialization AFAICT: https://github.com/zombocom/rack-timeout#rails-apps-manually

I was getting an error saying `service_timeout` was not an available method otherwise.